[ao] maintain BC for is_activation_post_process

HDCharles opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #89260

Summary: tests are failing due to code packaged with trained models calling now defunct function names (is_activation_post_process).

this diff maintains BC temporarily until the cached code can be refreshed

Test Plan: no functional change

Reviewers:

Subscribers:

Tasks:

Tags:

cc @jerryzh168 @jianyuh @raghuramank100 @jamesr66a @vkuzo @jgong5 @Xia-Weiwen @leslie-fang-intel

HDCharles committed:
[ao] maintain BC for is_activation_post_process

Summary: tests https://www.internalfb.com/mlhub/pipelines/runs/fblearner/388380008 and
and https://www.internalfb.com/tasks?t=135543814 are failing due to code packaged with trained models calling now defunct function names (is_activation_post_process).

this diff maintains BC temporarily until the cached code can be refreshed

Test Plan: buck2 run //aiplatform/modelstore/model_generation/flow/tests:cogwheel_trainer_gmpp_ig_test-launcher -- --build-fbpkg --run-disabled

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 reviewed (approved):
please fix lint

jerryzh168 commented:
also nit: probably need to create this from Phabricator and hide the internal links and test commands under "Facebook:" or "Meta:" (not sure which one works)

HDCharles committed:
Update on "[ao] maintain BC for is_activation_post_process"


Summary: tests https://www.internalfb.com/mlhub/pipelines/runs/fblearner/388380008 and
and https://www.internalfb.com/tasks?t=135543814 are failing due to code packaged with trained models calling now defunct function names (is_activation_post_process).

this diff maintains BC temporarily until the cached code can be refreshed

Test Plan: buck2 run //aiplatform/modelstore/model_generation/flow/tests:cogwheel_trainer_gmpp_ig_test-launcher -- --build-fbpkg --run-disabled

Reviewers:

Subscribers:

Tasks:

Tags:

cc jerryzh168 jianyuh raghuramank100 jamesr66a vkuzo jgong5 Xia-Weiwen leslie-fang-intel

[ghstack-poisoned]

HDCharles committed:
Update on "[ao] maintain BC for is_activation_post_process"


Summary: tests are failing due to code packaged with trained models calling now defunct function names (is_activation_post_process).

this diff maintains BC temporarily until the cached code can be refreshed

Test Plan: no functional change

Reviewers:

Subscribers:

Tasks:

Tags:

cc jerryzh168 jianyuh raghuramank100 jamesr66a vkuzo jgong5 Xia-Weiwen leslie-fang-intel

[ghstack-poisoned]

HDCharles commented:
@pytorchmergebot merge

DanilBaibak commented:
@pytorchbot revert -m "breaking internal builds" -c ghfirst

DanilBaibak commented:
Blocks the export of changes from the D41414219.
